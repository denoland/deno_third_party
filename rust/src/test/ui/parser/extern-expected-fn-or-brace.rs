// compile-flags: -Z parse-only

// Verifies that the expected token errors for `extern crate` are
// raised

extern "C" mod foo; //~ERROR expected one of `fn` or `{`, found `mod`
