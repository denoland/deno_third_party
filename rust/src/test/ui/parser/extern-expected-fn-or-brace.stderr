error: expected one of `fn` or `{`, found `mod`
  --> $DIR/extern-expected-fn-or-brace.rs:6:12
   |
LL | extern "C" mod foo; //~ERROR expected one of `fn` or `{`, found `mod`
   |            ^^^ expected one of `fn` or `{` here

error: aborting due to previous error

